src/Component/DataSource/Driver/Doctrine/ORM/ORMDriver.php

```diff
@@ -110,9 +110,10 @@ public function getResult(array $fields, ?int $first, ?int $max): Result
         $result = new ORMPaginator($query);
```

Comment:
Shouldn't this be simply called `$paginator` and then `$indexedResult` could be `$result`?